mono_repo/lib/src/commands/travis.dart

```diff
@@ -49,8 +66,11 @@ Future<Null> generateTravisConfig(
 
   _writeTravisYml(rootDirectory, sdks, envEntries, matrix);
 
-  _writeTravisScript(rootDirectory, _calculateTaskEntries(commandsToKeys),
-      _calculatePkgEntries(configs));
+  _writeTravisScript(
```

Comment:
Calling three methods w/ `prettyAnsi` starts to scream "make me a class" – but for later